Essentials/src/com/earth2me/essentials/ISettings.java

```diff
@@ -247,4 +249,12 @@
     boolean isSpawnOnJoin();
     
     boolean isTeleportToCenterLocation();
+    
+    boolean isCommandCooldownsEnabled();
+    
+    long getCommandCooldownMs(String label);
```

Comment:
It's not really a current cooldown, so technically there's nothing remaining. This command returns the consistent config variable.

Did you mean to comment about the IUser one instead?